valohai_yaml/objs/pipelines/override.py

```diff
@@ -0,0 +1,57 @@
+from collections import OrderedDict
+from typing import Iterable, List, Optional, Union
+
+from valohai_yaml.objs import Mount, Parameter
+from valohai_yaml.objs.base import Item
+from valohai_yaml.objs.environment_variable import EnvironmentVariable
+from valohai_yaml.objs.input import Input
+from valohai_yaml.objs.utils import check_type_and_dictify, check_type_and_listify, serialize_into
+from valohai_yaml.types import SerializedDict
+from valohai_yaml.utils.parse import parse_common_kwargs
+
+
+class Override(Item):
+    """Represents a Override class within an Execution Node."""
```

Comment:
```suggestion
    """Represents the step fields overridden in an execution or task node."""
```
might be more accurate?